dmutils/__init__.py

```diff
@@ -4,4 +4,4 @@
 import flask_featureflags  # noqa
 
 
-__version__ = '31.3.2'
+__version__ = '31.4.0'
```

Comment:
Is this actually a breaking change?

Reply:
I wasn't sure about this.

Reason it is a breaking change - someone who uses this function could see different behaviour from before
Reason it is not a breaking change - someone who uses this function will not need to make any code changes if they upgrade to this version as it somewhat adds functionality (support from another key)

https://semver.org/ refers to a breaking change as breaking the `public API` but that didn't help me decide either. My plan was to make it a breaking change when I withdraw support for the old token in the next PR.

I have this same debate with myself when things change in the FE toolkit. Let's say you changed a button from being small and blue to being big and green but no code changes are required for the person pulling in the repo, is that a breaking or non breaking change? The visual interface has debatably 'broken' but the interface might not have.

Would be good to hear your opinion?

Reply:
I would say non breaking change. All things being equal if I bumped the version of utils to this latests one, nothing will break. There's increased functionality (I can encode my tokens with two different keys now), but I don't need to change anything.

The example with the button is a bit different I think, as you now can't do thing you used to (click a small blue button).

I think major bump when you drop support for the old token